pppp/src/components/TimeInput.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div :class='className'>
+    <fieldset>
+      <legend class="time-legend">{{label}}</legend>
+      <div class="time-row">
+        <label v-bind:for="'hour-select' + label">Month:</label>
```

Comment:
Why is the label "month" here and further down?

Reply:
Good catch. I originally copied the component code from the `DateInput` component, so this was leftovers from the copy. I've pushed a fix for this.